CHANGELOG.md

```diff
@@ -3,6 +3,46 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.2
+
+_Jun 16, 2022_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🚀 Performance improvement for dynamic row height (#5135) @m4theushw
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.12.2` / `@mui/x-data-grid-pro@v5.12.2` / `@mui/x-data-grid-premium@v5.12.2`
```

Comment:
This is the first time. I did not updated the pickers version

Reply:
Ok, then we won't release them right? I'm asking because I thought they are automatically released but I guess if the version doesn't change then they won't be released.

Reply:
Since the date pickers have no synchronized version with the grid, it's okay to not release it. From v6 on, after we sync them, I think we should bump the version anyway, but that's a discussion beyond this issue, so we can reserve some time to think about how we're going to approach this situation in the future.

Reply:
The current strategy for synced packages is to not release if there is no change

Sometime you can see that we don't release the license package
And the next time we will bump it of two versions to re-establish the sync.

Reply:
The opposite may happen someday: no changes for the data grid but few changes for the date pickers. In this case, even the tags sequence will be broken if we don't bump them. For v6, data grid and date pickers will have to be bumped together.